Gemfile

```diff
@@ -15,7 +15,7 @@ gem 'aws-sdk', '~> 2.3.7'
 gem 'faraday', '~> 0.9.2'
 gem 'faraday_middleware', '~> 0.12.2'
 gem 'pundit', '~> 1.1.0'
-gem 'sdoc', '~> 0.4', group: :doc
+gem 'sdoc', '>= 1.1.0', group: :doc
```

Comment:
I don't even think this repo uses sdoc, can we remove it?
https://github.com/zooniverse/talk-api/search?q=sdoc

Reply:
Valid. I'm not aware of it being used, but this is also the first time I've touched this repo... the repo search does indeed seem to indicate that it is not in use, I will go ahead and remove.